content/applications/sales/crm/acquire_leads/opportunities_form.rst

```diff
@@ -0,0 +1,109 @@
+========================================
+Create opportunities from a contact form
+========================================
+
+Adding a contact form to a website makes it easy to convert visitors into leads and opportunities.
+After a visitor submits their information, an opportunity can be created automatically, and assigned
+to a designated sales team.
+
+.. _crm/customize-contact-form:
+
+Customize contact forms
+=======================
+
+By default, the *Contact Us* page on an Odoo website displays a preconfigured contact form. This
+form can be customized, as needed, with additional fields and settings to suit the needs of a
+specific sales team.
```

Comment:
I toyed with the idea of listing out some examples (eg- "Adding a number field and asking approximately how many employees a company has" or "providing the space for customers to enter their budget") but I was concerned that adding it before all of the information would draw attention away from the actual necessary/required info. I'd rather have them focus on the details they need instead of giving them examples of things they *could* do